src/main/java/frc/robot/Constants.java

```diff
@@ -3,6 +3,9 @@
 // the WPILib BSD license file in the root directory of this project.
 
 package frc.robot;
+import edu.wpi.first.networktables.NetworkTable;
+import edu.wpi.first.networktables.NetworkTableEntry;
+import edu.wpi.first.networktables.NetworkTableInstance;
```

Comment:
Please keep the code clean and remove any unnecessary imports from all files like these.  You can tell they are not necessary by the way VSCode highlights the import line.